js/libs/core.js

```diff
@@ -364,7 +364,7 @@ Fliplet.FormBuilder = (function() {
 
       component.props._componentsWithDescription = {
         type: Array,
-        default: ['flInput', 'flCheckbox', 'flRadio', 'flEmail', 'flNumber', 'flTelephone', 'flUrl', 'flTextarea', 'flWysiwyg', 'flSelect', 'flDate', 'flTime', 'flStarRating', 'flSignature', 'flImage', 'flFile']
```

Comment:
Why is it that the descriptions needed to be called in the component files as well? By removing them from the list, it looks like those 4 fields are not "componentsWithDescription", but they are -- it would cause future confusion and possibility of regression.